src/Converter/Feature2Mesh.js

```diff
@@ -9,8 +9,59 @@ import OrientationUtils from 'Utils/OrientationUtils';
 import Coordinates from 'Core/Geographic/Coordinates';
 
 const coord = new Coordinates('EPSG:4326', 0, 0, 0);
+
+class FeatureContext {
```

Comment:
Should the class be part of this file ? As it should also be used in Feature2Texture and in LabelLayer.
And is the name well choose as it's not the context of the feature (in itowns definition) but more of the featureGeometry with information of its feature and its featureCollection .

Reply:
I don't think style should be responsible for context.

Reply:
So should we create a new complet new file for it ?

Reply:
I agree, it should be in a separate file, and commented because it if is an argument of style properties functions (e.g. `base_altitude`) so its documentation would be useful to users